rust-test-program/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Is this file still needed?
I'd like to remove this from the PR and get the cargo build.rs version of the build process working instead.